### ISSUE TITLE
Question about loss function

### ISSUE THREAD

**Issue by Ivanyan0516** [megvii-model/YOLOF#56]
In the class loss, you use the sigmoid focal loss with the binary cross-entropy loss， why not choose the softmax focal loss? And what may be the reason why the multi-classes loss doesn't converge？
